values.yaml

```diff
@@ -355,7 +395,18 @@ st2garbagecollector:
   nodeSelector: {}
   tolerations: []
   affinity: {}
-
+  
+# Chatops service settings  
+st2chatops:
```

Comment:
I think would be useful to add `enabled: true/false` setting for the whole `st2chatops` as this is something optional, comparing to other required st2 core services.